app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/upload/UploadSuccessQrScreen.kt

```diff
@@ -27,114 +26,91 @@ import androidx.compose.material3.Surface
 
-    val scope = rememberCoroutineScope()
-
-    Column(
-        modifier = Modifier
-            .fillMaxSize()
-            .verticalScroll(rememberScrollState())
-            .padding(Margin.Medium),
-        horizontalAlignment = Alignment.CenterHorizontally,
-        verticalArrangement = Arrangement.Center,
-    ) {
-
-        Image(
-            imageVector = AppIllus.Beers.image(),
-            contentDescription = null,
-        )
+            Image(
+                imageVector = AppIllus.Beers.image(),
+                contentDescription = null,
+            )
 
-        Spacer(Modifier.height(Margin.Huge))
+            Spacer(Modifier.height(Margin.Huge))
 
-        Text(
-            text = stringResource(transferType.titleRes),
-            style = SwissTransferTheme.typography.h1,
-            color = SwissTransferTheme.colors.primaryTextColor,
-        )
+            Text(
+                text = stringResource(transferType.titleRes),
+                style = SwissTransferTheme.typography.h1,
+                color = SwissTransferTheme.colors.primaryTextColor,
+            )
 
-        Spacer(Modifier.height(Margin.Huge))
+            Spacer(Modifier.height(Margin.Huge))
 
-        QrCode(transferLink)
+            QrCode(transferLink)
 
-        transferType.descriptionRes?.let { descriptionRes ->
-            Spacer(Modifier.height(Margin.Huge))
-            Text(
-                text = stringResource(descriptionRes),
-                style = SwissTransferTheme.typography.bodyRegular,
-                textAlign = TextAlign.Center,
-                color = SwissTransferTheme.colors.secondaryTextColor,
-                modifier = Modifier.widthIn(max = Dimens.DescriptionWidth),
-            )
+            transferType.descriptionRes?.let { descriptionRes ->
+                Spacer(Modifier.height(Margin.Huge))
+                Text(
+                    text = stringResource(descriptionRes),
+                    style = SwissTransferTheme.typography.bodyRegular,
+                    textAlign = TextAlign.Center,
+                    color = SwissTransferTheme.colors.secondaryTextColor,
+                    modifier = Modifier.widthIn(max = Dimens.DescriptionWidth),
+                )
+            }
         }
-    }
 
-    // TODO: What do we want to do with this button placement?
-    LargeButton(
-        modifier = Modifier.padding(Margin.Medium),
-        style = ButtonType.SECONDARY,
-        titleRes = R.string.buttonCopyLink,
-        imageVector = AppIcons.DocumentOnDocument,
-        onClick = {
-            context.copyText(
-                text = transferLink,
-                showSnackbar = { scope.launch { snackbarHostState.showSnackbar(it) } },
-            )
-        },
-    )
+        ShareAndCopyButtons(
+            modifier = Modifier
+                .padding(horizontal = Margin.Medium)
+                .padding(bottom = Margin.Medium, top = Margin.Mini),
```

Comment:
in the case here where we already have to dissociate the padding, do we really want to double the padding() call ? 
I don't really know what I would prefer as both versions have their problem

Reply:
Chaining the paddings is pretty standard I think. This solution still has the benefit of conveying the idea that start and end are linked together which is really what I try to enforce on that line

Reply:
But I realized that the horizontal padding needs to go inside the component actually, so the paddings are not chained here anymore